### ISSUE TITLE
If I kill other players and he drops some ammo, sometimes I can't pick up except explosives.

### ISSUE THREAD

**Issue by cantianz** [redeclipse/base#1280]
This happens in master 2978 2.0.0 in Windows.

**Comment by cantianz** [comment]
I mean I always can pick up explosives but sometimes ammo can not even if I don't have full ammunitions.

**Comment by qreeves** [comment]
You can only pick up ammo for weapons in your loadout. Please check the ammo type.

**Comment by cantianz** [comment]
> 
> 
> You can only pick up ammo for weapons in your loadout. Please check the ammo type.

Yes I know, for example, I use a shotgun and sometimes I can't pick up ammo for shotgun,even if I don't have the maximum  ammunitions for shotgun. This will also happen when the Arena Mutator is enabled.

**Comment by qreeves** [comment]
Fix is related to same as one mentioned in #1527 and thus on hold temporarily.
